egui_glow/src/painter.rs

```diff
@@ -288,7 +226,7 @@ impl Painter {
 
     unsafe fn prepare_painting(
         &mut self,
-        gl_window: &glutin::WindowedContext<glutin::PossiblyCurrent>,
+        inner_size: [u32; 2],
```

Comment:
```suggestion
        [width_in_pixels, height_in_pixels]: [u32; 2],
```

makes it clearer what this is